sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/ParameterValidationInMethodShouldBeWrapped.cs

```diff
@@ -74,5 +74,18 @@ protected override sealed void Initialize(SonarAnalysisContext context)
                 },
                 RegisterSyntaxKinds);
         }
+
+        private static MethodDeclarationSyntax GetFirstEnclosingMethod(TRegisterExpressionSyntax registerExpression)
+        {
+            const int localFunctionStatementRawKind = 8830;
```

Comment:
Worth calling out that this is a workaround for syntax that Roslyn v1.3.2 can't handle.
Ideally, we want to be able to find all of this type of code easily so we can remove it when we move to a newer version of Roslyn - maybe we should have a well-known comment tag (e.g. _// UNSUPPORTED SYNTAX_), or perhaps we could put all of this type of code in a separate _UnsupportedSyntaxHelper_ class.

Also, it would be useful to comment in the method what version of Roslyn is required to support the syntax.

Reply:
+1 to extract the constant to a separate class and add comments.